### PR TITLE
[DSY-909] Feat/add size tokens

### PR DESCRIPTION
**Pull request by arielwb**
# Description

This PR adds the `size` token and its outputs to the `natds-themes` package.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit testing
- [x] 100% coverage

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

localhost_8080_theBodyShop_light html

### CODE REVIEW

File: packages/natds-themes/build/html/avon/dark.html [hunk 1]
```diff
@@ -14,6 +14,10 @@
       border: 1px solid #eaeaea;
       height: 50px;
     }
+
+    .size-sample {
```
Comment (jimmyandrade) [inline packages/natds-themes/build/html/avon/dark.html:18]: Curiosidade: o que seria esse `.size-sample`?

File: packages/natds-themes/build/ios/AvonSizes.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
+struct AvonSizes: Sizes {
+  let none: CGFloat = 0
+  let micro: CGFloat = 4
```
Comment (jimmyandrade) [inline packages/natds-themes/build/ios/AvonSizes.swift:3]: Outra curiosidade: esses valores viram `dp` depois?

File: packages/natds-themes/config/ios/__file_snapshots__/registerStructFormat.snap.swift [hunk 3]
```diff
@@ -0,0 +1,3 @@
+struct Struct: Interface {
+  let primaryType = #F091C9
```
Comment (jimmyandrade) [inline packages/natds-themes/config/ios/__file_snapshots__/registerStructFormat.snap.swift:2]: Curiosidade: podemos usar sem aspas no Swift?

Comment (arielwb) [reply packages/natds-themes/config/ios/__file_snapshots__/registerStructFormat.snap.swift:2]: acho que não! esse é um snapshot que testa o template, talvez falte algo no teste 🤔

File: packages/natds-themes/build/ios/AvonSizes.swift (hunk 2 above)
Comment (arielwb) [reply packages/natds-themes/build/ios/AvonSizes.swift:3]: esses valores são do tipo CGFloat (o nosso bom e velho 0.1) e são independentes de unidade de medida de tela.
@raonivaladares algum complemento?

File: packages/natds-themes/build/html/avon/dark.html (hunk 1 above)
Comment (arielwb) [reply packages/natds-themes/build/html/avon/dark.html:18]: é uma classe com as definições especificas para os blocos de tokens `size` para o sample html (to seguindo o padrão color-sample, size-sample) pois são blocos com "samples" dos tokens. Faz sentido?

Comment (jimmyandrade) [reply packages/natds-themes/build/html/avon/dark.html:18]: Faz sim. Obrigado :D

File: packages/natds-themes/config/ios/__file_snapshots__/registerStructFormat.snap.swift (hunk 3 above)
Comment (jimmyandrade) [reply packages/natds-themes/config/ios/__file_snapshots__/registerStructFormat.snap.swift:21]: @arielwb como ficou essa questão? É sem aspas mesmo?

**Pull request merged**